guides/araujo/FTP Writeup.md

```diff
@@ -0,0 +1,92 @@
+---
+
+
+---
```

Comment:
You may not see it yet but there is a thing called front matter which jekyll uses and it will need to be in your guide for the site to work. Use
```
---
layout: post
title: box
author: name
---
```